ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.txt

```diff
@@ -11,9 +11,116 @@ hideall "--sync--"
+1105.2 "Sanguine Clot x3" sync / 03:........:Added new combatant Sanguine Clot\./
+1108.3 "Viscous Clot x2" sync / 03:........:Added new combatant Viscous Clot\./
+1120.0 "Viscous Clot x3" sync / 03:........:Added new combatant Viscous Clot\./
+1125.2 "Sanguine Clot x4" sync / 03:........:Added new combatant Sanguine Clot\./
+1127.4 "Bozjan Soldier x2" sync / 03:........:Added new combatant Bozjan Soldier\./
+1132.2 "Viscous Clot" sync / 03:........:Added new combatant Viscous Clot\./
+1144.2 "Viscous Clot x3" sync / 03:........:Added new combatant Viscous Clot\./
+1156.2 "Viscous Clot x3" sync / 03:........:Added new combatant Viscous Clot\./
+1162.1 "Viscous Clot" sync / 03:........:Added new combatant Viscous Clot\./
+1165.4 "Sanguine Clot x2" sync / 03:........:Added new combatant Sanguine Clot\./
+1168.2 "Viscous Clot x2" sync / 03:........:Added new combatant Viscous Clot\./
+1172.7 "Sanguine Clot x4" sync / 03:........:Added new combatant Sanguine Clot\./
+1177.4 "Bozjan Soldier x2" sync / 03:........:Added new combatant Bozjan Soldier\./
+1192.2 "Viscous Clot" sync / 03:........:Added new combatant Viscous Clot\./
+1204.2 "Viscous Clot x2" sync / 03:........:Added new combatant Viscous Clot\./
+1225.1 "Sanguine Clot" sync / 03:........:Added new combatant Sanguine Clot\./
+1227.2 "Bozjan Soldier x2" sync / 03:........:Added new combatant Bozjan Soldier\./
+1233.1 "Sanguine Clot x4" sync / 03:........:Added new combatant Sanguine Clot\./
+1277.3 "Bozjan Soldier x2" sync / 03:........:Added new combatant Bozjan Soldier\./
+1297.3 "Grim Reaper Enrage" sync / 03:........:Added new combatant Grim Reaper\./ window 300,10
+1302.3 "Death Scythe" #sync /:Grim Reaper:5747:/
+1305.3 "Death Scythe" #sync /:Grim Reaper:5747:/
+1308.3 "Death Scythe" #sync /:Grim Reaper:5747:/
+1311.3 "Death Scythe" #sync /:Grim Reaper:5747:/
+1314.4 "Death Scythe" #sync /:Grim Reaper:5747:/
+1317.4 "Death Scythe" #sync /:Grim Reaper:5747:/
+1320.4 "Death Scythe" #sync /:Grim Reaper:5747:/
+1323.5 "Death Scythe" #sync /:Grim Reaper:5747:/
+1326.5 "Death Scythe" #sync /:Grim Reaper:5747:/
+1327.3 "Bozjan Soldier x2" sync / 03:........:Added new combatant Bozjan Soldier\./
+1329.5 "Death Scythe" #sync /:Grim Reaper:5747:/
+1332.6 "Death Scythe" #sync /:Grim Reaper:5747:/
+1335.6 "Death Scythe" sync /:Grim Reaper:5747:/ window 5,10
+1338.6 "Death Scythe" #sync /:Grim Reaper:5747:/
+1341.7 "Death Scythe" #sync /:Grim Reaper:5747:/ window 10,10
+1341.7 "Death Scythe" #sync /:Grim Reaper:5747:/
+1344.7 "Death Scythe" #sync /:Grim Reaper:5747:/
+1347.7 "Death Scythe" #sync /:Grim Reaper:5747:/
+1350.8 "Death Scythe" #sync /:Grim Reaper:5747:/
+1353.8 "Death Scythe" #sync /:Grim Reaper:5747:/
+1356.8 "Death Scythe" #sync /:Grim Reaper:5747:/
+1359.8 "Death Scythe" #sync /:Grim Reaper:5747:/
+
+
+### Golems (The Granary, right side)
+2000.0 "--sync--" sync / 22:........:Bicolor Golem:........:01/ window 2000,0
```

Comment:
Oh! It's starting late because I messed this line up.  I think it needs to be:
`2000.0 "--sync--" sync / 22:........:Bicolor Golem:........:Bicolor Golem:01/ window 2000,0`